go.mod

```diff
@@ -1,5 +1,7 @@
-module github.com/OneSignal/onesignal-go-api/v2
+module github.com/OneSignal/onesignal-go-api
```

Comment:
Any insights on this change? Not sure if this is correct

Reply:
Hmm not sure, but looking at the history, there's 2 PRs about this:
- https://github.com/OneSignal/onesignal-go-api/pull/31
- https://github.com/OneSignal/onesignal-go-api/pull/36

Reply:
There's also this https://github.com/OneSignal/onesignal-go-api/pull/32

Reply:
Ah, there's a PR sitting in the upstream repo to fix this: https://github.com/OneSignal/API-Client-Libraries/pull/161

Reply:
Looks fixed now